src/Microsoft.Sbom.Api/Executors/JsonSerializationStrategyFactory.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Sbom.Api.Utils;
+
+namespace Microsoft.Sbom.Api.Workflows.Helpers;
+
+public static class JsonSerializationStrategyFactory
```

Comment:
Let's make `JsonSerializationStrategyFactory`, `Spdx3SerializationStrategy`, `Spdx2SerializationStrategy`, and `IJsonSerializationStrategy` all `internal` unless we need them for unit tests. They shouldn't be part of the public API.
```suggestion
internal static class JsonSerializationStrategyFactory
```